tests/kubeless-test-client/test-kubeless.go

```diff
@@ -229,36 +266,45 @@ func ensureOutputIsCorrect(host, expectedOutput, testID, namespace, testName str
 		for {
 			select {
 			case <-timeout:
-				log.Println("Timeout: Check if virtual service has been created")
+				log.Printf("[%v] Timeout: Check if virtual service has been created", testName)
```

Comment:
Lets get the exact virtualservice for the function which we expect to be there rather than all. This will  facilitate debugging in logs in api-controller.